### ISSUE TITLE
Get response from commands 

### ISSUE THREAD

**Issue by ghost** [enniel/ami#15]
Hi @enniel, I don't know if I'm doing something wrong or is just an issue.

I'm running the next code:
```
$algo = Artisan::call('ami:action', [
    'action' => 'ConfbridgeList',
    '--arguments' => [
        'ActionID' => '1',
        'Conference' => $extension
    ]
]);

dd($algo);
```
If the conference exist $algo equals 0 and if the conference don't exist the program never reach the dd($algo) statement.

Running the command version in the CLI I get the response succes if the conference exists, but when the conference don't exist I get nothing returned :(.

P.D: Sorry for my english, trying to improve 👍

**Comment by ghost** [comment]
Pls, some help?, what I'm doing bad.

**Comment by rnrstar** [comment]
I'm having the same issue. 

I have in a controller:
```
public function test()
    {
        $result = Artisan::call('ami:cli', [
    		'command'      => 'queue show',
    		'--autoclose' => true,
]);
dd($result);
    }
```

The output is always 0. When I do it at the command line it works:

php artisan ami: cli 'queue show' --autoclose

The above command generates the expected output.

**Comment by rnrstar** [comment]
It looks like the output of 0 is correct in that Artisan::call is only going to return true or false depending on the success of the command.

**Comment by ghost** [comment]
Yes, it looks like output 0 is correct, but when an error occurs, it never reaches the dd ($ result) statement.

So what to do?, I need to know if the action was ok or not.

Thx.

**Comment by peymannavard** [comment]
I have the same that problem
always output show 0
    public function Outcall (){

         $result = Artisan::call('ami:action', [
             'action' => 'Command',
             '--arguments' => [
                 'Command' => "queue show"
             ]
         ]
);

         print_r($result);

How to fix this ?

**Comment by peymannavard** [comment]
Nobody help ?

**Comment by komilgulboev** [comment]
The same issue still in Laravel 5.7, query taking too long and failed again of timeout. 
Using below script for check:
```
 $result = Artisan::call('ami:cli', [
    		'command'      => 'queue show',
    		'--autoclose' => true,
]);
return response()->json($result);
```
But test from cli is working well, from cli tested

>  php artisan ami:listen
